test/unit/SpinViewer/SpinViewer.spec.js

```diff
@@ -1,12 +1,28 @@
 import SpinViewer from "../../../src/SpinViewer/SpinViewer";
 
 describe("SpinViewer", function() {
+	describe("constructor", () => {
+		it("should accept with no option", () => {
+			// Given
+			var target = sandbox();
+			target.innerHTML = `<div"></div>`;
+
+			// When, no option
+			let inst = new SpinViewer(target);
```

Comment:
This is not a real use case. If you want to increase the test coverage, it's better to remove uncovered logic branch in the source code.